api/k8s/v1/k8s.proto

```diff
@@ -273,12 +273,23 @@ message UpdateDeploymentRequest {
 message UpdateDeploymentResponse {
 }
 
+// This message type is used to add support for nullable strings and is an
+// alternative to the well-known `StringValue` type. We need it, because the
+// grpc-gateway used by Clutch deserializes a null `StringValue` as an empty
+// string.
+message NullableString {
+  oneof kind {
+    google.protobuf.NullValue null = 1;
+    string value = 2;
+  }
+}
```

Comment:
not clear on why this diverges from the way that string value is written i.e.
```
message StringValue {
  // The string value.
  string value = 1;
}
```

also would prefer we make whatever solution it is global and copy all of the wrappers over to `clutch/api/api/v1/wrappers.proto`.

Reply:
I went for this approach in order to make it very explicit that the value is nullable and how to express a null value.

I don't however have a strong preference for the current version vs copying the wrapper types to Clutch — if you prefer the latter let me know and I'll update the PR.

Reply:
Yeah, though the wrapper types are poorly named (`NullableString` is a better name for sure), I don't want to introduce a new concept when the core proto ecosystem already has one. Simply copying the wrapper types into our own file to circumvent the pbjson flattening is to me the best fix and clearly communicates our intent.

in documenting that file we can refer to https://github.com/protobufjs/protobuf.js/issues/677 and https://github.com/protocolbuffers/protobuf-go/blob/3f7a61f89bb6813f89d981d1870ed68da0b3c3f1/encoding/protojson/well_known_types.go#L35-L44